web/pages/group/[...slugs]/index.tsx

```diff
@@ -398,66 +408,11 @@ function GroupOverview(props: {
             />
           </Col>
         )}
-
-        <Col className={'mt-2'}>
-          <div className="mb-2 text-lg">Members</div>
-          <GroupMemberSearch members={members} group={group} />
-        </Col>
```

Comment:
So we are removing the list of members? I think that's ok for now.

Reply:
Yep, Stephen and Ian said that was fine.